docs/docs/deployment/oss/deployment-options/kubernetes/managing-pod-eviction.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Handling pod eviction'
+description: How to handle pod eviction in Kubernetes deployments.
+sidebar_position: 4400
+---
+
+When running Dagster in Kubernetes environments, pod evictions can occur due to resource constraints or node scaling operations. To prevent runs from hanging indefinitely, you can implement the following solutions:
```

Comment:
```suggestion
When Dagster is run in Kubernetes environments, pod evictions can occur due to resource constraints or node scaling operations. To prevent runs from hanging indefinitely, you can implement the following solutions:
```